src/ontology/mondo-edit.obo

```diff
@@ -465244,171 +465409,6 @@ intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"}
 intersection_of: MONDO:0700097 MONDO:0005150 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog age-related macular degeneration
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
 
-[Term]
-id: MONDO:1010021
```

Comment:
@sabrinatoro can you tell me how exactly this term was added?

Reply:
it was added using a ROBOT template.

Reply:
The `sh run.sh make merge-template` workflow?

Reply:
added using ROBOT template in this PR: https://github.com/monarch-initiative/mondo/pull/6607

Reply:
yes, using  sh run.sh make merge-template workflow